images/linux/scripts/helpers/get-toolset-value.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+################################################################################
+##  File:  toolset.sh
+##  Desc:  Helper functions to retrive toolset data
+################################################################################
+
+toolset_path="/imagegeneration/installers/toolset.json"
```

Comment:
We shouldn't use this approach with file for every helper function.
Let's just add function to existing helper https://github.com/actions/virtual-environments/blob/main/images/linux/scripts/helpers/install.sh that is imported almost everywhere.